src/main/java/de/unistuttgart/overworldbackend/repositories/PlayerNPCStatisticRepository.java

```diff
@@ -0,0 +1,18 @@
+package de.unistuttgart.overworldbackend.repositories;
+
+import de.unistuttgart.overworldbackend.data.PlayerNPCStatistic;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface PlayerNPCStatisticRepository extends JpaRepository<PlayerNPCStatistic, UUID> {
+  List<PlayerNPCStatistic> findPlayerNPCStatisticByLectureId(int lectureID);
+  Optional<PlayerNPCStatistic> findPlayerNPCStatisticByNpcIdAndLectureIdAndPlayerstatisticId(
```

Comment:
Are you trying to make the method name unreadable?
```suggestion
  List<PlayerNPCStatistic> findByLectureId(int lectureID);
  Optional<PlayerNPCStatistic> findByNpcIdAndLectureIdAndPlayerstatisticId(
```
Also, `NPC` is a multi-word identifier, so it should be written `NPC` as we agreed.

Reply:
Names have to be that way in a repository becauese of the way a repository finds the columns in a table

Reply:
??
Since when?
Why would you need the class inside the method name?

Reply:
Its just possible when the attribute of the class PlayerNPCStatistic is named `nPC`. We will keep it npc in this pr.